easybuild/easyconfigs/t/torchvision/torchvision-0.5.0-fosscuda-2019b-Python-3.7.4-PyTorch-1.4.0.eb

```
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'torchvision'
+version = '0.5.0'
+versionsuffix = '-Python-%(pyver)s-PyTorch-1.4.0'
+
+homepage = 'https://github.com/pytorch/vision'
+description = " Datasets, Transforms and Models specific to Computer Vision"
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+
+source_urls = ['https://github.com/pytorch/vision/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['eb9afc93df3d174d975ee0914057a9522f5272310b4d56c150b955c287a4d74d']
+
+builddependencies = [('CMake', '3.15.3')]
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('Pillow-SIMD', '6.0.0'),
+    ('PyTorch', '1.4.0', '-Python-%(pyver)s'),
+]
+
+use_pip = True
+download_dep_fail = True
+
```

There is a new requirement for PythonPackages to use the `sanity_pip_check `
```suggestion
sanity_pip_check = True

```